src/SFA.DAS.AssessorService.Application.Api/Controllers/CertificateController.cs

```diff
@@ -107,5 +107,15 @@ public async Task<IActionResult> Approvals([FromBody] CertificateApprovalRequest
 
             return Ok();
         }
+
+        [HttpPost("prepareprivatecertificatesforbatchrun", Name = "PreparePrivateCertificatesforBatchRun")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(Certificate))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> PreparePrivateCertificatesforBatchRun([FromBody] PrivateCertificatePrepareForBatchRunRequest privateCertificatePrepareForBatchRunRequest)
+        {
+            await _mediator.Send(privateCertificatePrepareForBatchRunRequest);
```

Comment:
The await is outside of the return OK() - is this by design?

Reply:
yep there's no response with it.